elements/lisk-db/src/utils.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+export const formatInt = (num: number | bigint): string => {
+	let buf: Buffer;
+	if (typeof num === 'bigint') {
+		buf = Buffer.alloc(8);
+		buf.writeBigInt64BE(num);
+	} else {
+		buf = Buffer.alloc(4);
+		buf.writeInt32BE(num);
+	}
+	return buf.toString('binary');
+};
+
+export const getStartPrefix = (prefix: string): string => `${prefix}\x00`;
```

Comment:
also, this can be changed to `getPrefix` if we change the other one to `getSuffix`

Reply:
But it's not getting prefix or suffix.
It more like getting first key possible of the prefix, and getting last key possible of the prefix

Reply:
the `getLastPrefix` sounds more correct

Reply:
`getFirstPrefix` and `getLastPrefix `